src/main/java/core/basesyntax/service/FruitService.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.dto.FruitRecordDto;
+import core.basesyntax.service.impl.Operation;
+import java.util.List;
+import java.util.Map;
+
+public interface FruitService {
+    void saveData(List<FruitRecordDto> parsedLines,
+                  Map<Operation, FruitOperationHandler> operationStrategy);
```

Comment:
```suggestion
```
don't take strategy as input, initialize it as class field

Reply:
Done.

Reply:
> don't take strategy as input, initialize it as class field

this comment is not fixed